src/app/ReadHandler.h

```diff
@@ -68,7 +68,7 @@ class ReadHandler : public Messaging::ExchangeDelegate
      *  of the object and releases all held resources.
      *
      */
-    void Shutdown();
+    void Shutdown(bool aAbort = false);
```

Comment:
A comment about what 'abort' means would be helpful. As mentioned above, I  believe some constants would be clearer than a bool (kAbortExchange seems obvious to read)

Reply:
done, thanks